erpnext/hr/doctype/employee_loan/employee_loan.json

```diff
@@ -237,7 +238,7 @@
    "label": "Status", 
    "length": 0, 
    "no_copy": 0, 
-   "options": "Loan Unpaid\nLoan Paid\nEMI in progress\nLoan Repaid", 
+   "options": "Loan Unpaid\nLoan Disbursed\nPartial Loan Disbursed", 
```

Comment:
Status can be "Sanctioned", "Partially Disbursed", "Fully Disbursed", "Repaid/Closed"